crypto/addresses/src/lib.rs

```diff
@@ -121,6 +151,17 @@ impl TryFrom<u8> for Version {
     }
 }
 
+impl ToString for Version {
```

Comment:
ToString shouldn’t be implemented directly: [Display](https://doc.rust-lang.org/std/fmt/trait.Display.html) should be implemented instead, and you get the ToString implementation for free.

Reply:
replaced all ToString to Display throughout the entire codebase